cds_ils/config.py

```diff
@@ -417,15 +417,15 @@ def _parse_env_bool(var_name, default=None):
         },
         "unit:institution": {
             "elasticsearch": "keyword",
-            "marshmallow": List(SanitizedUnicode()),
+            "marshmallow": SanitizedUnicode(),
         },
         "unit:project": {
             "elasticsearch": "keyword",
             "marshmallow": SanitizedUnicode(),
         },
         "unit:study": {
             "elasticsearch": "keyword",
-            "marshmallow": List(SanitizedUnicode()),
+            "marshmallow": SanitizedUnicode(),
```

Comment:
@kprzerwa do you bless this change?

Reply:
This change was motivated by the answer from the librarians `Library:There is no KB in CDS yet.` can be checked in [CodiMD](https://codimd.web.cern.ch/uR0k-4yiTzW0BSnp7mDw0Q#Institutions)

Reply:
Sorry I don't follow, they said that there is no KB yet, so you change here from List to string?
I'm afraid I do not bless this change (neither the previous one, with the institutions), @ntarocco do you know anything more about this that I am not aware of?

IMO it should be a list, what if the record is connected with more than one institution or study ? It can happen, like in case of experiments and accelerators